src/angularjs/src/app/index.route.js

```diff
@@ -9,32 +9,32 @@
     function routerConfig($stateProvider, $urlRouterProvider) {
         $stateProvider
             .state('login', {
-                url: '/login/',
+                url: '/admin/login/',
```

Comment:
Since login would log the user into the entire application, maybe this should continue to live at `/login`